libs/knowledge-store/ragstack_knowledge_store/graph_store.py

```diff
@@ -243,19 +250,24 @@ def __init__(
             """
         )
 
+    def table_name(self) -> str:
+        """Returns the fully qualified table name."""
+        return f"{self._keyspace}.{self._node_table}"
+
     def _apply_schema(self) -> None:
         """Apply the schema to the database."""
         embedding_dim = len(self._embedding.embed_query("Test Query"))
         self._session.execute(f"""
-            CREATE TABLE IF NOT EXISTS {self._keyspace}.{self._node_table} (
+            CREATE TABLE IF NOT EXISTS {self.table_name()} (
                 content_id TEXT,
                 kind TEXT,
                 text_content TEXT,
                 text_embedding VECTOR<FLOAT, {embedding_dim}>,
 
                 link_to_tags SET<TUPLE<TEXT, TEXT>>,
                 link_from_tags SET<TUPLE<TEXT, TEXT>>,
-                metadata_blob TEXT,
+                attributes_blob TEXT,
```

Comment:
changed to the naming used in cassio

Reply:
I don't love mixing the `attributes` and `metadata` prefixes. It seems like this is *just* writing the metadata a few different ways...

Reply:
reverted to old name